src/components/no-options/no-options.jsx

```diff
@@ -2,6 +2,9 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styles from './no-options.css';
 
+/**
+ * @deprecated
```

Comment:
Wait, why is this one deprecated? Are we moving away from this component for Multi and Single Select?

Reply:
I think this is leftover from moving code around. I was thinking this was more of an private (but shared) component. At a glance, it's documentation page does not provide that much usefulness / utility. I was thinking of removing the documentation page.